docs/edgekv_cli.md

```diff
@@ -381,6 +392,34 @@ Usage:
 | - | - | - |
 | tokenName | required | token name |
 
+### List Permission Groups
+
+List the permission groups with EdgeKV access
+
+Usage:
+`edgekv list auth-groups [options]`
+
+Example:
+`edegkv list auth-groups --groupId gid1,gid2 --include_ew_groups`
+
+| Option | Description |
+| - | - |
+| --groupIds | List the EdgeKV access capabilities for the specified permission groups, separated by a comma |
+| --include_ew_groups | List all permission groups with EdgeKV or EdgeWorkers access capabilities |
+| -h, --help  | Display information on how to use this EdgeKV command |
+
+### Modify Permission Group
+
+Modify the permission group associated with the namespace
+
+Usage:
+`edgekv modify auth-group <namespaceId> <groupId>`
+
+| Argument | Existence | Description |
+| - | - | - |
+| namespaceId | required | Namespace identifier |
+| groupid | required | Group identifier |
```

Comment:
Suggest modifying this to "Permissions group Identifier" to avoid confusion with the data group identifier. Please check with Sarah for final text.